### PR TITLE
feat: add LO Statement of Facts form.

### PR DESCRIPTION
**Pull request by benoitdevos**
logion-network/logion-internal#440

### CODE REVIEW

File: src/loc/statement/PathModel.ts [hunk 1]
```diff
@@ -51,58 +62,114 @@ export const DEFAULT_PATH_MODEL: PathModel = {
...
+    value(name: string) {
+        return this.params[name];
     }
 
-    return `?${params.join("&")}`;
+    valueAsString(name: string): string {
+        return decodeURIComponent(this.params[name] as string);
+    }
+
+    valueAsFloat(name: string): number {
```
Comment (gdethier) [inline src/loc/statement/PathModel.ts:152]: Why not `valueAsNumber`?

File: src/loc/statement/StatementOfFactsSummary.tsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+                <strong>signed version of your Statement of Facts</strong> before closing the related LOC. DO NOT put
+                the signed version of this Statement of Facts in another LOC, otherwise, the LOC URL mentioned in the
+                Statement of Facts will be not valid.
+            </p>
+            <p>
+                For your convenience, all files mentioned in the Statement of Facts are also ready to be downloaded by
+                clicking the related icon hereafter.
+            </p>
+            <Row>
+                <Col>Statements od Facts</Col>
```
Comment (gdethier) [inline src/loc/statement/StatementOfFactsSummary.tsx:48]:
```suggestion
                <Col>Statements of Facts</Col>
```

File: src/loc/statement/StatementOfFactsSummary.tsx [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+        <>
+            <h3>Statement of Facts generator</h3>
+            <p>Your statement of Facts is ready, you can view it by clicking on the related icon, hereafter.</p>
+            <p>
+                <strong>As a mandatory step, you must submit the signed Statement of Fact PDF in the related LOC:
+                </strong>
+                You can access the related LOC by clicking on the link, hereafter. Do not forget to upload the
+                <strong>signed version of your Statement of Facts</strong> before closing the related LOC. DO NOT put
+                the signed version of this Statement of Facts in another LOC, otherwise, the LOC URL mentioned in the
+                Statement of Facts will be not valid.
```
Comment (gdethier) [inline src/loc/statement/StatementOfFactsSummary.tsx:41]:
```suggestion
                Statement of Facts will not be valid.
```

**Review by gdethier — APPROVED** [review]
Great and thanks for the refactoring!

File: src/loc/statement/StatementOfFactsSummary.tsx (hunk 2 above)
Comment (benoitdevos) [reply src/loc/statement/StatementOfFactsSummary.tsx]: Done.

File: src/loc/statement/StatementOfFactsSummary.tsx (hunk 3 above)
Comment (benoitdevos) [reply src/loc/statement/StatementOfFactsSummary.tsx]: Done.

File: src/loc/statement/PathModel.ts (hunk 1 above)
Comment (benoitdevos) [reply src/loc/statement/PathModel.ts]: Agreed.

**Pull request merged**